scripts/anonymization/README.md

```diff
@@ -0,0 +1,220 @@
+
+In the following example, the amount of bins is now 1000 and the lower and upper bounds are given by hand such that no preprocessing epsilon must be spent!
+
+```xml
+<continuousConfig>
+    <column name="i_price" bins="1000" lower="2.0" upper="100.0">
+<continuousConfig>
+```
+
+The default parameters must be added inside of the `<column>`-tag. (See below)
+
+| Name  | Default | Possible Value    | Description                                 |
+| ----- | ------- | ----------------- | ------------------------------------------- |
+| name  | -       | Any string        | The name of the column                      |
+| bins  | 10      | Any value         | The amount of distinct values in the output |
+| lower | -       | Any value         | Lower bound of the values                   |
+| upper | -       | Any value > lower | Upper bound of the values                   |
+
+---
+
+A full working example could look like this:
+
+```xml
+<anonymization>
+    <table name="item" epsilon="1.0" pre_epsilon="0.0" algorithm="aim">
+        <!-- Column categorization -->
+        <droppable>i_id</droppable>
+        <categorical>i_name,i_data,i_im_id</categorical>
+        <continuous>i_price</continuous>
+        <!-- Continuous column fine-tuning -->
+        <continuousConfig>
+            <column name="i_price" bins="1000" lower="2.0" upper="100.0">
+        <continuousConfig>
+    </table>
+</anonymization>
+```
+
+### Sensitive Values
+
+The anonymization module supports value faking to handle sensitive values. A `<sensitive>`-tag must be created to signify that column faking should be applied. Column faking can be defined individually on each column. To do so, parameters can be added inside of the `<column>`-tag. (See below)
+
+| Name    | Default | Possible Value                                   | Description                                                                                    |
+| ------- | ------- | ------------------------------------------------ | ---------------------------------------------------------------------------------------------- |
+| name    | -       | Any string                                       | The name of the column                                                                         |
+| method  | pystr   | Any method provided by the python faker library  | The faking method of the library. If the method is not found, a random string will be inserted |
+| locales | -       | A string of supported locales separated by comma | The locale to produce localized values like chinese street names or english first names        |
+| seed    | 0       | Any integer value                                | The privacy budget spent on preprocessing of the data. Only necessary for continuous columns   |
+
+```xml
+<anonymization>
+    <table name="item" algorithm="aim">
+        <sensitive>
```

Comment:
Is "column faking" mutually exclusive with "differential privacy"?

Or can the schema somehow be unified with the one I mentioned above?

Reply:
We would like to handle it separately as part of the Anonymization.

This allows for anonymization of sensitive columns without differential privacy (For example for the Twitter workload, user_profiles table)

or coupled anonymization (Run DP first and then handle the sensitive values for higher privacy guarantees)